packages/web-react/src/components/Pill/Pill.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ActionColors, EmotionColors } from '../../constants';
+import { Pill } from './Pill';
+import ReadMe from './README.md';
+
+const meta: Meta<typeof Pill> = {
+  title: 'Components/Pill',
+  component: Pill,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    color: {
+      control: 'select',
+      options: [...Object.values(ActionColors), ...Object.values(EmotionColors), 'selected', 'unselected'],
+      table: {
+        defaultValue: { summary: 'selected' },
+      },
+    },
+    elementType: {
```

Comment:
There is some weird stuff in the description… 🤔 

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/dfa952cf-fa29-4e85-b035-f28f72c1f189)

Reply:
This is auto generated from the code. We can override it in the argtypes, but that's another place to edit everything. I would like to avoid that. We can look in the future into how we set the type and why it does that.